src/main/resources/operations/getFailedResults.xml

```diff
@@ -1,11 +1,133 @@
+                                            <authorizationCode>
+                                                <refreshToken>{$ctx:uri.var.refreshToken}</refreshToken>
+                                                <clientId>{$ctx:uri.var.clientId}</clientId>
+                                                <clientSecret>{$ctx:uri.var.clientSecret}</clientSecret>
+                                                <tokenUrl>{$ctx:uri.var.tokenUrl}</tokenUrl>
+                                                <authMode>Header</authMode>
+                                            </authorizationCode>
+                                        </oauth>
+                                    </authentication>
+                                </http>
+                            </endpoint>
+                        </call>
+                    </then>
+                    <else/>
+                </filter>
+            </then>
+            <else>
+                <property name="uri.var.clientId" expression="get-property('clientId')"/>
+                <property name="uri.var.clientSecret" expression="get-property('clientSecret')"/>
+                <property name="uri.var.refreshToken" expression="get-property('refreshToken')"/>
+                <property name="uri.var.tokenUrl" expression="get-property('tokenUrl')"/>
+                <call>
+                    <target contentType="application/json" type="body"/>
+                    <endpoint>
+                        <http method="get" uri-template="{uri.var.url}">
+                            <timeout>
+                                <duration>60000</duration>
+                                <responseAction>fault</responseAction>
+                            </timeout>
+                            <suspendOnFailure>
+                                <initialDuration>2000</initialDuration>
+                                <progressionFactor>1.0</progressionFactor>
+                                <maximumDuration>3000</maximumDuration>
+                            </suspendOnFailure>
+                            <authentication>
+                                <oauth>
+                                    <authorizationCode>
+                                        <refreshToken>{$ctx:uri.var.refreshToken}</refreshToken>
+                                        <clientId>{$ctx:uri.var.clientId}</clientId>
+                                        <clientSecret>{$ctx:uri.var.clientSecret}</clientSecret>
+                                        <tokenUrl>{$ctx:uri.var.tokenUrl}</tokenUrl>
+                                        <authMode>Header</authMode>
+                                    </authorizationCode>
+                                </oauth>
+                            </authentication>
+                        </http>
+                    </endpoint>
+                </call>
+            </else>
+        </filter>
+        <filter regex="200" source="$axis2:HTTP_SC">
```

Comment:
What will happen when outputtype is json and  there is a failure?

Reply:
By default, saleforce bulk api sends failure messages in application/json format.